patterns/2-structured/release-process.md

```diff
@@ -0,0 +1,64 @@
+## Title
+
+Standard Release Process
+
+## Patlet
+
+Teams are reluctant to use InnerSource projects that they are unfamiliar with when they are unclear about the release process.
+Providing clear release notes and a published artifact gives people confidence you are publishing a quality product.
+
+## Problem
+
+When a team is deciding whether to use an InnerSource project, one of their considerations is whether they trust that they can rely on the given project for an extended period. Switching the tools/projects that they are using has a cost, so they only want to make those investments when necessary and has tangible benefits.
+
+It is common practice for Open Source projects to have versioned releases, with notes documenting breaking changes, and new features along with either a published binary or link to a Docker image. This practice may not be as transparent or well documented/visible for InnerSource projects, modules, etc.
+
+InnerSource projects that don't have a published artifact or release process reduces trust. Teams won't know when they can expect the next release, when breaking changes are introduced, etc.
+
+## Context
+
+Large organizations produce a lot of internal software, much of which could be reused by teams across the company. Operational tooling, software libraries, and Infrastructure as a Service modules are all examples of this. Most large organizations don't publish internal software to be consumed by other teams in the company, however. This can occur either because they are too busy provide documentation or don't realize it's value to others.
```

Comment:
```suggestion
Large organizations produce a lot of internal software, much of which could be reused by teams across the company. Operational tooling, software libraries, and Infrastructure as a Service modules are all examples of this. Most large organizations don't publish internal software to be consumed by other teams in the company, however. This can occur either because they are too busy to provide documentation or don't realize it's value to others.
```

Does the "it's" in the last sentence refer to the documentation or to the project?

Reply:
Refers to the project - I'll try to make that more clear